### PR TITLE
[MIG] auth_user_case_insensitive: Migration to 12.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

**Comment by ghost** [comment]
What should I do about CLA problem? I am not associated with Mr. Salmon, or LasLabs.

**Comment by pedrobaeza** [comment]
@Retropikzel don't worry about that complaint, as it's a gap in our bot.

Please fix Travis however.

**Comment by remytms** [comment]
@pedrobaeza The only error I can see in the travis check is this one:
```
ERROR openerp_test odoo.addons.auth_ldap.models.res_company_ldap: An LDAP exception occurred: {'desc': "Can't contact LDAP server"}
```

I'm not sure that this error is linked to the `auth_user_case_insensitive`. What do you think ?

**Review by remytms — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Yeah, that one is the problem. You have to use `mute_logger` in that test for avoiding the log complaint, although you check that the raise is happening.

**Comment by remytms** [comment]
I will try to look at this tomorrow at OCA sprint.

**Comment by remytms** [comment]
Hi @pedrobaeza ,
Hi @Retropikzel ,

I have fixed the travis error with a `mute_logger` as suggested. See this commit https://github.com/coopiteasy/server-auth/commit/f67ab2f54a0da0208656acf5a2020d80a254206d.

@Retropikzel I have created a PR to your branch https://github.com/Retropikzel/server-auth/pull/1.

**Review by remytms — APPROVED** [review]
Looks good to me. :)

**Comment by remytms** [comment]
@pedrobaeza Travis check are fixed. :) Thanks for your hints.

**Comment by pedrobaeza** [comment]
You can now look for a second reviewer reviewing you other PRs and ask for it in exchange

**Review by jarmokortetjarvi — APPROVED** [review]

**Comment by jarmokortetjarvi** [comment]
Looks good, and tested the logic on runbot with different spellings of admin

**Comment by pedrobaeza** [comment]
Well, reviews come from non PSC members, but everything green + the hope you become more active in OCA :) get me to request the merge: /ocabot merge

**Comment by ghost** [comment]
@pedrobaeza Thank you. :)

File: auth_user_case_insensitive/tests/test_res_users.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        self.assertEqual(
+            self.login.lower(), rec_id.login,
+            'Login was not lowercased when saved to db.',
+        )
+
+    def test_login_login_is_lowercased(self):
+        """ It should verify the login is set to lowercase on login """
+        rec_id = self._new_record()
+        # We have to commit this cursor, because `_login` uses a fresh cursor
+        self.env.cr.commit()
```
Comment (yajo) [inline auth_user_case_insensitive/tests/test_res_users.py:53]: Oops... this passed unnoticed? A commit in a test is a time bomb... :scream: :bomb:

**Pull request merged**